### ISSUE TITLE
dependency problem with kwin-lowlatency?

### ISSUE THREAD

**Issue by slartibart70** [ZaWertun/fedora-copr-kde5#61]
Hi,
i was using kwin-lowlatency and tried to update to latest repo contents.
I got some nasty dependency problems like so

```
Problem 1: package plasma-workspace-xorg-5.20.4-3.fc33.x86_64 requires kwin-x11 >= 5.20, but none of the providers can be installed
  - package kwin-x11-5.20.3-1.fc33.x86_64 requires kwin-common(x86-64) = 5.20.3-1.fc33, but none of the providers can be installed
[..]
```
A revert to kwin fixed all of this, updates are ok now.
Still, installing kwin-lowlatency will downgrade lots of libs again..

```
Downgrading:
 libkworkspace5                             x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde           113 k
 plasma-lookandfeel-fedora                  noarch          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde           1.1 M
 plasma-workspace                           x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde           6.7 M
 plasma-workspace-common                    x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde            29 k
 plasma-workspace-geolocation               x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde            55 k
 plasma-workspace-geolocation-libs          x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde            23 k
 plasma-workspace-libs                      x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde           2.1 M
 plasma-workspace-wayland                   x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde            29 k
 plasma-workspace-xorg                      x86_64          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde            35 k
 sddm-breeze                                noarch          5.20.4-1.fc33           copr:copr.fedorainfracloud.org:zawertun:kde           936 k
```

Is there something you can do to fix those problems?

**Comment by ZaWertun** [comment]
Got it. I'll fix this dependency.

**Comment by ZaWertun** [comment]
Should be fixed in plasma-workspace-5.20.5.

**Comment by felipelutz** [comment]
It worked. Thank you!
